### DISCUSSION TITLE
macOS guest temporarily freezes on M1

### DISCUSSION THREAD

**Discussion by ghost** [utmapp/UTM#3933] [UTM Help (macOS)]
UTM 3.15
Guest: macOS
M1 iMac

Hello, I'm running a macOS guest on UTM 3.15 on an M1 iMac. The macOS Monterey guest will freeze for maybe 10 secs and then the VM will work again. If I connect to the VM through remote desktop, it is the same behavior.

**Comment by lnguyen234** [comment]
It could be something with the display resolution, try set the Resolution of the vm (UTM -> VM configuration -> Display) to something lower (eg 1024x640)

**Comment by misterjame** [comment]
I adjusted the resolution to 1024x640 and still encounter the problem. I am using a VPN inside the VM, not sure if that is related

**Reply by orichter** [reply]
I had a similar problem. In my case it almost always came as a result of using the scroll function on my Mac Mouse and/or trackpad. Reducing the mouse "tracking speed" on my host and guest seems to have mostly solved the problem. I also minimized usage of the scroll functions of the mouse.

**Comment by stevenbousamra** [comment]
Not buying it.

Version 4.3.5 also freezes all aspects of the guest (Desktop / Activity monitor is frozen / no scrolling / no input from keyboard) for a period of approximately 20 seconds within the first 5 minutes of start / wakeup.  The host has no impact, shows nothing large processing, or the like.

Am running a Mac Studio M1 Ultra with a 2TB hdd with less than 1 TB consumed.  Memory is below 90 GB, with only 16.5 GB consumed by the applicaiton

**Comment by naufalfachrian** [comment]
Any update on this?
